vars/perfRegressionParallelPipeline.groovy

```diff
@@ -248,7 +254,7 @@ def call(Map pipelineParams) {
                                                         if [[ ! -z "${params.scylla_ami_id}" ]] ; then
                                                             export SCT_AMI_ID_DB_SCYLLA=${params.scylla_ami_id}
                                                         elif [[ ! -z "${supportedVersions}" ]]; then
-                                                            export SCT_SCYLLA_VERSION=${supportedVersions.last()}
+                                                            export SCT_SCYLLA_VERSION=${supportedVersions}
```

Comment:
I think this would still gonna fail, since it's not define in the scope of the split to a different runner

Reply:
i have no idea what it means...

here (https://github.com/scylladb/scylla-cluster-tests/pull/6707#issuecomment-1777827188) you can see both kind of jobs running